src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -313,13 +314,34 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
     const quoteData = q as QuoteTypeMap[typeof type];
     const flashbots = (SwapSettings.flashbots.value && inputAsset.chainId === ChainId.mainnet) ?? false;
 
+    const isNativeWrapOrUnwrap =
+      isWrapEth({
+        buyTokenAddress: quoteData.buyTokenAddress,
+        sellTokenAddress: quoteData.sellTokenAddress,
+        chainId: inputAsset.chainId,
+      }) ||
+      isUnwrapEth({
+        buyTokenAddress: quoteData.buyTokenAddress,
+        sellTokenAddress: quoteData.sellTokenAddress,
+        chainId: inputAsset.chainId,
+      });
+
+    // Do not deleeeet the comment below 😤
+    // About to get quote
     const parameters: Omit<RapSwapActionParameters<typeof type>, 'gasParams' | 'gasFeeParamsBySpeed' | 'selectedGasFee'> = {
       sellAmount: quoteData.sellAmount?.toString(),
       buyAmount: quoteData.buyAmount?.toString(),
       chainId: inputAsset.chainId,
       assetToSell: inputAsset,
       assetToBuy: outputAsset,
-      quote: quoteData,
+      quote: {
+        ...quoteData,
+        buyAmountDisplay: isNativeWrapOrUnwrap ? quoteData.buyAmount : quoteData.buyAmountDisplay,
+        sellAmountDisplay: isNativeWrapOrUnwrap ? quoteData.sellAmount : quoteData.sellAmountDisplay,
```

Comment:
what's the logic for overriding these two? I don't think it's necessary.

Reply:
This was done on the BX so figured it was necessary. If it's not I can remove it

Reply:
I can't find anything like that here https://github.com/rainbow-me/browser-extension/blob/ec53546df9bc6ed3adb902d48dc719053ba0e298/src/entries/popup/pages/swap/SwapReviewSheet/SwapReviewSheet.tsx#L293-L304. Am I looking at the wrong place?